chart/templates/deployment.yaml

```diff
@@ -52,13 +52,13 @@ spec:
           livenessProbe:
             httpGet:
               path: /health
-              port: 10999
+              port: 8080
```

Comment:
On port 8080 is not the distributor responding but the argo service itself... I am not sure it's correct to use that as a distributor liveness/readiness probe

Reply:
I saw that `prometheus-service` does the same and at the time I didn't think it was a big deal, now though I was thinking that if for whatever reason the service has a problem we are tearing down also the distributor and that doesn't seem right to me. As a counter-example in the `job-executor-service` the probes have been removed when upgrading to keptn 0.12 (see keptn-contrib/job-executor-service#173). 
@christian-kreuzberger-dtx  any thoughts/preferences/guidelines here ?

Reply:
This is the best practice as defined by Keptn core:
distributor uses health check of port 8080, which is usually the service itself.

I agree that looks weird. i reached out to the Keptn team back then, and they said it's okay.

Reply:
@christian-kreuzberger-dtx Thank you for letting us know. Then I will leave the health checks as they are now.